app/src/components/Form/Form.js

```diff
@@ -4,18 +4,33 @@ import { Formik } from 'formik';
 import * as Yup from 'yup';
 import _ from 'lodash';
 
-export const formFromFields = (fields, formikProps) =>
-  Object.keys(fields).map(id =>
-    React.createElement(fields[id].component, {
+export const formFromFields = (fields, formikProps, dynamicRequire) =>
+  Object.keys(fields).map(id => {
+    const x = !!(
+      dynamicRequire[id] || _.get(fields[id], 'validation._exclusive.required')
+    );
+    const y = !!formikProps.values[id];
+    if (x && !y) console.log('isrequired', id, x, y);
```

Comment:
This whole x and y section is just for testing, yay?

Reply:
Yes is now removed.